templates/shared/pricing/_kubernetes-consulting.html

```diff
@@ -24,9 +20,9 @@ <h2 class="p-heading--four">Kubernetes Explorer</h2>
   <div class="col-6 p-card">
     <div class="p-card__header">
       <h2 class="p-heading--four">Kubernetes Discoverer</h2>
-      <p><span class="p-heading--two">$45,000</span> one off fee</p>
+      <p><span class="p-heading--two">$45,000</span><br />Workshop and custom architecture</p>
     </div>
-    <p>Get up and running in one week with a customised architecture to fit your requirements, including deploying on virtualised environments, private and public clouds.</p>
+    <p>Week-long training and workshop together with design of a customised architecture to fit your requirements, including deploying on virtualised environments, private and public clouds.</p>
```

Comment:
There's a bit missing at the bottom: `Optional add on: AI/ML $40,000 one off fee`. Not sure if it should be removed in the copydoc or included here